src/modules/tasksList.js

```diff
@@ -0,0 +1,3 @@
+const tasksList = [
+];
```

Comment:
Since you are just exporting the array, you could kindly use:

`export default [];`